web/src/main/java/org/springframework/security/web/DefaultRedirectStrategy.java

```diff
@@ -41,6 +44,9 @@ public class DefaultRedirectStrategy implements RedirectStrategy {
 
 	private boolean contextRelative;
 
+	@Nullable
+	private HttpStatus statusCode;
```

Comment:
Instead of setting this to `null`, please set it to 302.

Reply:
Got it, but actually, `HttpStatus.FOUND`, right?